taskwiki/util.py

```diff
@@ -88,7 +88,8 @@ def tw_args_to_kwargs(args):
         elif arg.startswith('+'):
             value = arg[1:]
             # Ignore virtual tags
-            if not value.isupper():
+            vtags = ["BLOCKED", "UNBLOCKED", "BLOCKING", "DUE", "DUETODAY", "TODAY", "OVERDUE", "WEEK", "MONTH", "QUARTER", "YEAR", "ACTIVE", "SCHEDULED", "PARENT", "CHILD", "UNTIL", "WAITING", "ANNOTATED", "READY", "YESTERDAY", "TOMORROW", "TAGGED", "PENDING", "COMPLETED", "DELETED", "UDA", "ORPHAN", "PRIORITY", "PROJECT", "LATEST"]
```

Comment:
Would be nice to include a script to generate this list. E.g.

```
curl --silent --show-error --fail --location https://github.com/GothenburgBitFactory/taskwarrior/raw/2.6.0/src/commands/CmdInfo.cpp | grep -P -o 'virtualTags \+= "\K\w+'
```

```
curl --silent --show-error --fail --location https://github.com/GothenburgBitFactory/taskwarrior/raw/2.6.0/doc/man/task.1.in | awk '/^ +([A-Z]+) +Matches if the task/ { print $1 }'
```

Unfortunately the lists don't match as taskwarrior developer's aren't very diligent in keeping them in sync despite all the comments reminding them to do so. :-(

I also think this might better go into constants.py, and should be accompanied by a test. :-)

Reply:
In constants you can do something like:

```
TASKWARRIOR_VIRTUAL_TAGS = """
BLOCKED
UNBLOCKED
…
""".split()
```

to make it easier to paste the output of that command.

Regarding the test: not necessary to add a new test, augmenting an existing one should be good enough I think. :-)